src/CognitiveServices/CognitiveServices.Test/ScenarioTests/CognitiveServicesTestRunner.cs

```diff
@@ -0,0 +1,83 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Management.CognitiveServices;
+using System.Collections.Generic;
+using Microsoft.Azure.Management.Internal.Resources;
+using RestTestFramework = Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using Microsoft.Azure.Management.Network;
+using Microsoft.Azure.Commands.TestFx;
+using Xunit.Abstractions;
+
+namespace Microsoft.Azure.Commands.Management.CognitiveServices.Test.ScenarioTests
+{
+    public class CognitiveServicesTestRunner
+    {
+        protected readonly ITestRunner TestRunner;
+
+        protected CognitiveServicesTestRunner(ITestOutputHelper output)
+        {
+            TestRunner = TestManager.CreateInstance (output)
+                .WithNewPsScriptFilename ($"{GetType().Name}.ps1")
+                .WithProjectSubfolderForTests ("ScenarioTests")
+                .WithCommonPsScripts (new[]
+                {
+                    @"Common.ps1",
+                    @"../AzureRM.Resources.ps1",
+                })
+                .WithNewRmModules (helper => new[]
+                {
+                    helper.RMProfileModule,
+                    helper.RMNetworkModule,
+                    helper.GetRMModulePath("Az.CognitiveServices.psd1")
+                })
+                .WithNewRecordMatcherArguments (
+                    userAgentsToIgnore: new Dictionary<string, string>
+                    {
+                        {"Microsoft.Azure.Management.Resources.ResourceManagementClient", "2016-02-01"},
+                    },
+                    resourceProviders: new Dictionary<string, string>
+                    {
+                        {"Microsoft.Resources", null},
+                        {"Microsoft.Features", null},
+                        {"Microsoft.Authorization", null},
+                        {"Microsoft.Network", null}
+                    }
+                ).WithManagementClients(
```

Comment:
WithManagementClients method can be removed. Please make sure all tests can pass after removing it